llvm/lib/Transforms/Scalar/LoopIdiomRecognize.cpp

```diff
@@ -1154,7 +1154,7 @@ bool LoopIdiomRecognize::processLoopStridedStore(
                                               PatternValue, ".memset_pattern");
       GV->setUnnamedAddr(
           GlobalValue::UnnamedAddr::Global); // Ok to merge these.
-      GV->setAlignment(Align(PatternArgTy->getPrimitiveSizeInBits()));
+      GV->setAlignment(Align(PatternArgTy->getPrimitiveSizeInBits() / 8));
```

Comment:
Just above you already queried the getTypeSizeInBits for PatternArgTy, avoid repeating it? Every getPrimitiveSizeInBits makes me nervous

Reply:
The logic to create a GlobalVariable isn't needed any more, so I've now dropped this whole bit.